exercises/practice/palindrome-products/.articles/config.json

```
@@ -0,0 +1,11 @@
+{
+    "articles": [
+        {
+            "uuid": "987cf2d1-7225-436f-9228-b88120ae2e70",
+            "slug": "is-palindrome-check",
+            "title": "Is a palindrome number check",
+            "blurb": "A few ways to check if a number is a palindrome.",
```

Minor tweak:

```suggestion
            "blurb": "Different ways to check if a number is a palindrome.",
```

This makes it slightly more appealing I feel.